app/src/main/java/com/plan2meet/dto/GroupDTO.kt

```diff
@@ -2,6 +2,7 @@ package com.plan2meet.dto
 
 class GroupDTO (
     var calendarDTO    : CalendarDTO?,
+    //make sure to use your variables in your code to lessen technical debt.
```

Comment:
Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.